packages/legend-query-builder/src/stores/explorer/QueryBuilderPropertySearchState.ts

```diff
@@ -182,8 +244,11 @@ export class QueryBuilderPropertySearchState {
     });
   }
 
+  //takes in the searched mapped property and retrieves the toggles that the
```

Comment:
Let's use block comment instead of line comments for documenting method. In VSCode, you can try https://stackoverflow.com/questions/34220564/is-there-a-way-to-generate-jsdoc-comments-in-visual-studio-code